### DISCUSSION TITLE
Customize changelog with emojis

### DISCUSSION THREAD

**Discussion by baggiponte** [commitizen-tools/commitizen#679] [Q&A] [answered]
Hello! I would like to basically add emojis to the changelog generated with commitizen (e.g [*à la* polars](https://github.com/pola-rs/polars/releases/tag/py-0.16.10). How can I do that? I looked at the docs but it wasn't super clear. Thanks!

EDIT:

solved, it's the `change_type_map` parameter [here](https://commitizen-tools.github.io/commitizen/customization/#1-customize-in-configuration-file).

**Comment by woile** [comment, marked as answer]
Yes, you can use the `change_type_map` to customize it.

**Reply by baggiponte** [reply]
Unrelated, can I edit the changelog format too? I would like to prepend a `Release v$version` - that will be useful for using gh actions to generate release notes [like PDM does](https://github.com/pdm-project/pdm/blob/ef4969477e830d4dd138599428d481f9d5130f20/.github/workflows/release.yml#LL56)

**Reply by woile** [reply]
Mmm I think that would add some problems :/ 

Hopefully [this PR](https://github.com/commitizen-tools/commitizen/pull/645) will address it.
